squidb/src/com/yahoo/squidb/data/SquidDatabase.java

```diff
@@ -175,7 +175,7 @@ protected void onTablesCreated(SQLiteDatabaseWrapper db) {
      * @param oldVersion the current database version
      * @param newVersion the database version being upgraded to
      * @return true if the upgrade was handled successfully, false otherwise
-     * @see {@link #onMigrationFailed(MigrationFailedException)}
+     * @see onMigrationFailed(MigrationFailedException)
```

Comment:
When I do this Android Studio gives me an error. What was the issue you were having with it?